numpy/ma/core.py

```diff
@@ -6397,6 +6397,20 @@ def __repr__(self):
             # it's a subclass, or something is wrong, make it obvious
             return object.__repr__(self)
 
+    def __format__(self, format_spec):
+        # Replace ndarray.__format__ with the default, which supports no format characters.
+        # Supporting format characters is unwise here, because we do not know what type
+        # the user was expecting - better to not guess.
+        try:
+            return object.__format__(self, format_spec)
+        except Exception:
```

Comment:
I think we know that `ValueError` is expected, so that would be cleaner (e.g. the super unlikely case that someone hits `ctrl+c` at exactly this time.

Reply:
Looks like we a get a `TypeError`, not a `ValueError`, e.g.:

```
In [23]: object.__format__(123, '<5')                                                                                                                                       
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-23-8ccbea622452> in <module>
----> 1 object.__format__(123, '<5')

TypeError: unsupported format string passed to int.__format__
```